### ISSUE TITLE
Feature Request - set Editor `\page` lines to `position:sticky;`

### ISSUE THREAD

**Issue by Gazook89** [naturalcrit/homebrewery#2038]
Another 'pie-in-the-sky' feature request:   Change the `\page` lines in the editor to stick to the top of the editor as you scroll, disappearing when scrolling to the next page.

**Comment by Gazook89** [comment]
I got the below to work just using DevTools and screwing with the CSS of a few elements.  However, since CodeMirror only renders a certain number of lines at a time, the CSS created in devTools is wiped if I scroll too far out of view.

I would be willing to actually work on this but just want to see what others think before doing anything further.  If there isn't interest I will close out the Issue.

![image](https://media.giphy.com/media/y9uRw4GNBGtH6fQsMb/giphy.gif)

**Comment by calculuschild** [comment]
I think could have it's uses but we'd want to fine tune it.
For one, the only benefit we currently get is the page number stays visible, but even then it's really hard to see. We'd probably want to add some kind of labeling ability to the \page line that users would actually want to keep visible.

Tldr; I think it's cool, but we probably need to come up with some more useful reason for it to be there.

**Comment by Gazook89** [comment]
Okay I’m going to close this issue then— if ever we come back to it, it’s here.
